server/tests/audit_math/test_bravo.py

```diff
@@ -479,35 +479,37 @@ def test_compute_risk_empty(contests):
 
 # Useful test data
 round0_sample_results = {
-    "test1": {"cand1": 0, "cand2": 0,},
-    "test2": {"cand1": 0, "cand2": 0, "cand3": 0,},
-    "test3": {"cand1": 0,},
-    "test4": {"cand1": 0,},
-    "test5": {"cand1": 0, "cand2": 0,},
-    "test6": {"cand1": 0, "cand2": 0, "cand3": 0},
-    "test7": {"cand1": 0, "cand2": 0, "cand3": 0},
-    "test8": {"cand1": 0, "cand2": 0, "cand3": 0},
-    "test9": {"cand1": 0, "cand2": 0, "cand3": 0},
-    "test10": {"cand1": 0, "cand2": 0, "cand3": 0},
-    "test11": {"cand1": 0, "cand2": 0},
-    "test12": {"cand1": 0, "cand2": 0, "cand3": 0},
-    "test_small_third_candidate": {"cand1": 0, "cand2": 0, "cand3": 0},
+    "test1": {"round0": {"cand1": 0, "cand2": 0,}},
+    "test2": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0,}},
+    "test3": {"round0": {"cand1": 0,}},
+    "test4": {"round0": {"cand1": 0,}},
+    "test5": {"round0": {"cand1": 0, "cand2": 0,}},
+    "test6": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
+    "test7": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
+    "test8": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
+    "test9": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
+    "test10": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
+    "test11": {"round0": {"cand1": 0, "cand2": 0}},
+    "test12": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
+    "test_small_third_candidate": {"round0": {"cand1": 0, "cand2": 0, "cand3": 0}},
```

Comment:
should these all just be `None` now?

Reply:
Ah, yep. It doesn't really matter, but I will fix it.